protocol/xmrtaker/swap_state.go

```diff
@@ -438,9 +438,8 @@ func (s *swapState) setXMRMakerKeys(sk *mcrypto.PublicKey, vk *mcrypto.PrivateVi
 }
 
 // lockETH the Swap contract function new_swap and locks `amount` ether in it.
-// TODO: rename to lockAsset or createNewSwap or something?
 // TODO: update units to not necessarily be an EtherAmount
-func (s *swapState) lockETH(amount common.EtherAmount) (ethcommon.Hash, error) {
+func (s *swapState) lockAsset(amount common.EtherAmount) (ethcommon.Hash, error) {
```

Comment:
The initial comment line needs the function name updated. It also looks like it needs the verb "calls".